lib/services.template.ejs

```diff
@@ -20,8 +20,7 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
     var m = url.match(/^(?:https?:)?\/\/([^\/]+)/);
     return m ? m[1] : null;
   }
-
-  var urlBaseHost = getHost(urlBase) || location.host;
+  var urlBaseHost = getHost(urlBase) ? urlBase : location.host;
```

Comment:
yes that is intentional as per discussion with @persimmons on #251. This is for the proxy situation where you need to differentiate between multiple loopback servers on the same base URL.